app/modules/transcribe/transcribe.controller.js

```diff
@@ -93,12 +94,7 @@ function TranscribeController($stateParams, $modal, $scope, $window, Annotations
         vm.subject = SubjectsFactory.current;
     }
 
-    function subjectLoadError(result) {
-        if (result === 'outOfData') {
-            $scope.$broadcast('subject:outOfData');
-        } else {
-            console.error('Error loading subject', result);
-        }
+    function handleErrors(result) {
+        return Promise.reject(new Error('Error loading subject'));
```

Comment:
Why does this return a promise, and raise a new error? Should it display the actual error that occurred too?